### PR TITLE
as I said, bosses out

### PR DESCRIPTION
**Pull request by mandresm**
I'm not sure I want my bosses around in a party

@ukrebska @vasmue @mathanase @antoniofis10000

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,9 @@ The shiniest party lights you have ever seen
 Nikolay's latest fancy plots 
 
 ## Blacklist (list)
+
+Thomas J.
+Gerrit L.
```
Comment (mandresm) [inline README.md:54]: Gerrit is cool, let's invite him

**Comment by rjuhrbandt** [comment]
I agree with @mandresm

File: README.md (hunk 1 above)
Comment (valentinludwig) [reply README.md:54]: Fair enough. Let's hope TJ is not on GitHub.

Comment (VeraBullfinch) [reply README.md:54]: The bosses watch you

**Pull request merged**